src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackageExtractionContext.cs

```diff
@@ -15,7 +15,7 @@ public class PackageExtractionContext
         /// </summary>
         public bool UseLegacyPackageInstallPath { get; set; }
 
-        public PackageSaveMode PackageSaveMode { get; set; } = PackageSaveMode.Nupkg;
+        public PackageSaveMode PackageSaveMode { get; set; } = PackageSaveMode.Nupkg | PackageSaveMode.Files;
```

Comment:
If the Default value is relevant to a specific scenario can you make it clear by adding a v3 suffix.
Also having a v2 default constant would be beneficial.